packages/evm/src/clients/testnet-client.ts

```diff
@@ -36,7 +36,7 @@ import { centrifugeTestnet } from "./custom/centrifuge";
  * Testnet chains
  */
 export const TESTNET_CHAINS = {
-  "ethereum-sepolia": sepolia,
+  "eth-sepolia": sepolia,
```

Comment:
i think we still use "ethereum-sepolia" for testnet, no? if you need another one for devnet, perhaps creating a new one instead of replacing this?